egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -194,6 +221,23 @@ def PrintConfig(file_name, config_lines):
     f.write("\n".join(config_lines['component-nodes']))
     f.close()
 
+def ParseCnnString(cnn_string):
```

Comment:
@vijayaditya , sorry that i don't quite understand how i should modify this, could you please specific how you want the user to input the CNN parameters ?